src/components/Input/BoolInput.tsx

```diff
@@ -1,35 +1,76 @@
 import { JSX } from 'react';
-import { Checkbox, Group } from '@mantine/core';
+import { Checkbox, Group, MantineStyleProps } from '@mantine/core';
 
-import { Label } from '../Label/Label';
+import { InfoBox } from '@/components/InfoBox/InfoBox';
+import { RequireAtLeastOne } from '@/types/types';
 
-interface Props {
-  name: string;
-  description: string | JSX.Element;
-  value: boolean;
-  setValue: (value: boolean) => void;
-  label?: JSX.Element;
+interface BaseProps extends MantineStyleProps {
+  onChange?: (value: boolean) => void;
+  info?: string | JSX.Element;
   disabled?: boolean;
 }
 
+// Either the label or the ariaLabel must be provided, or both
+interface Label {
+  label?: string;
+  ariaLabel?: string;
+}
```

Comment:
```suggestion
}

```

Reply:
Hm, I goruped them intentionally to show that they are connected